src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -464,5 +472,16 @@ public void keyPressed(java.awt.event.KeyEvent e) {
                 }
             }
         }
+
+        /**
+         * Clears the search bar and select first entry, if available
+         *
+         * @author Aly Mohamed
```

Comment:
Please do not add `@author` tags even if the exercise sheet requested it.

Reply:
Will do in the next pull request based on javafxGlobalEverything branch.

Reply:
Just remove it on this PR. Then, this PR is ready for merge.

Please create a new branch based on `javafxGlobalEverything`. I think, the implementation will differ much, so you can start from scratch? 😇

Reply:
Of course, no problem :smiley: I'm actually enjoying this myself and am learning a lot about developing larger-scale open-source projects so starting from scratch sounds fun :smile:.